packages/charts/chart-utilities/src/PlotlySchemaConverter.ts

```diff
@@ -1,11 +1,12 @@
-import type { Datum, TypedArray, PlotData, PlotlySchema } from './PlotlySchema';
+import type { Datum, TypedArray, PlotData, PlotlySchema, Data } from './PlotlySchema';
 import { decodeBase64Fields } from './DecodeBase64Data';
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export interface OutputChartType {
   isValid: boolean;
   errorMessage?: string;
   type?: string;
+  validDataIndices?: number[];
```

Comment:
>number[]; [](http://example.com/codeflow?start=21&length=9)

should we change this to a map of [index, chartType] for future use so we dont have to change the contract as we start supporting composite traces

Reply:
Wouldn't the chart be of a single type, even if multiple traces can be plotted? For example, a VerticalBarChart for a bar and line combination.

Reply:
There could be a dashboard with line and donut for example.